docs/cover.md

```diff
@@ -1,57 +1,71 @@
-# Code coverage for GoAWK
 
-GoAWK implements code coverage functionality, similar to the one built in the Golang itself (https://go.dev/blog/cover).
+# GoAWK's code coverage feature
 
-![screenshot](cover.png)
+GoAWK implements code coverage functionality, similar to the [code coverage built into Go](https://go.dev/blog/cover). This feature was implemented by Volodymyr Gubarkov ([xonixx](https://github.com/xonixx)), primarily in [PR #154](https://github.com/benhoyt/goawk/pull/154).
 
-## How to use
+Here is a screenshot using GoAWK's coverage feature on a simple AWK program (`prog.awk`):
+
+![Example code coverage screenshot](cover.png)
```

Comment:
I'm open to other options, but I'd prefer a simple program like this one in the screenshot, rather than a complicated one like we had before.

Reply:
I agree.